staff/asier/tasks-list_express/src/data/tasksData.js

```diff
@@ -0,0 +1,150 @@
+const tasks = []
+
+const users = []
+
+let taskId = 0
+
+const tasksData = {
+
+    /* /// create task /// */
+
+    create(text, username) {
+        
+        if (!users.includes(username)) {
+            users.push(username)
+        }
+
+        taskId += 1
+        tasks.push({ taskId, text, username, done: false })
+    },
+
+    /* /// List every task /// */
+
+    list() {
+        if (tasks.length < 1) throw Error('There are no tasks to show')
+
+        return tasks
+    },
+
+    retrieve(id) {
+        const task = tasks.find(task => task.taskId === Number(id))
+
+        if (task) return task
+
+        throw Error('Task does not exist.')
+    },
+
+    /*  /// Remove task from tasklist */
+
+    delete(id) {
+        const index = tasks.findIndex(x => x.taskId === Number(id))
```

Comment:
I would change param 'x' for 'task' for better syntax purpose

Reply:
You right! Changed... :+1: